src/components/renders/bootstrap/select/select.e2e.ts

```diff
@@ -39,4 +39,11 @@ describe("gx-select", () => {
     await page.waitForChanges();
     expect(await page.find("select")).toHaveClass("foo-class");
   });
+
+  it("should set the input with suggest list if suggest attr is enabled", async () => {
+    const input = await element.find("input[list='gx-select-auto-id-0']");
```

Comment:
I think that using the `id` makes this test too brittle and that the id is an internal implementation detail.
We could achieve the same result without referencing the `id` and by checking that the `list` attribute of the `input` element and the `id` of the `datalist` element are equal.

Reply:
I updated that e2e, check it in 5709850